compiler.pri

```diff
@@ -260,8 +260,23 @@ unix:!macx {
 		QMAKE_LFLAGS *= -Wl,--no-add-needed
 	}
 
-	QMAKE_CFLAGS *= -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2
-	QMAKE_CXXFLAGS *= -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2
+	# Ensure _FORTIFY_SOURCE is not used in debug builds.
```

Comment:
Might want to link to https://sourceware.org/bugzilla/show_bug.cgi?id=13979 or some suitable reference as it's not intuitive why it has to be that way.